base/compiler/ssair/slot2ssa.jl

```diff
@@ -617,7 +618,11 @@ function construct_ssa!(ci::CodeInfo, ir::IRCode, sv::OptimizationState,
             end
             continue
         end
+
         @timeit "liveness" (live = compute_live_ins(cfg, slot))
+        for block in live.live_in_bbs
+            push!(live_slots[block], idx)
+        end
         for li in live.live_in_bbs
             cidx = findfirst(x::TryCatchRegion->x.leave_block==li, catch_entry_blocks)
```

Comment:
```suggestion
        for li in live.live_in_bbs
            push!(live_slots[li], idx)
            cidx = findfirst(x::TryCatchRegion->x.leave_block==li, catch_entry_blocks)
```
maybe merge these two loops?